cmd/kubeadm/app/util/config/testdata/conversion/internal.yaml

```diff
@@ -0,0 +1,161 @@
+API:
```

Comment:
do these need to be fully populated? It makes it difficult to see what's actually being tested.

Reply:
Yes, these are roundtrip tests, so the same data is everywhere, but we test conversion and make sure there is no data loss etc.